doc/development.md

```diff
@@ -0,0 +1,287 @@
+* [Licenses management](#licenses-management)
+  * [Adding manually a new license](#adding-manually-a-new-license)
+
+## Prerequisites
+
+* [operator-sdk] version v0.8.0
+* [git][git_tool]
+* [go] version 1.12.5+
+* [kubernetes] version v1.13.0+
+* [oc] version v4.1+
+* Access to a Openshift v4.1.0+ cluster.
+* A user with administrative privileges in the OpenShift cluster.
+
+## Clone repository
+
+```sh
+mkdir -p $GOPATH/src/github.com/3scale
+cd $GOPATH/src/github.com/3scale
+git clone https://github.com/3scale/3scale-operator
+cd 3scale-operator
+git checkout master
+```
+
+## Building 3scale operator image
+
+[Clone the repo](#clonerepository)
+
+Build operator image
+
+```sh
+make build IMAGE=quay.io/myorg/3scale-operator VERSION=test
+```
+
+## Run 3scale Operator
+
+### Run 3scale Operator Locally
+
+Run operator from command line, it will not be deployed as pod.
+
+* [Clone the repo](#clonerepository)
+
+* Register the 3scale-operator CRDs in the OpenShift API Server
+
+```sh
+// As a cluster admin
+for i in `ls deploy/crds/*_crd.yaml`; do oc create -f $i ; done
+```
+
+* Create a new OpenShift project (optional)
+
+```sh
+export NAMESPACE=operator-test
+oc new-project $NAMESPACE
+```
+
+* Run operator
+
+```sh
+make local
+```
+
+### Deploy 3scale Operator Manually
+
+Build operator image and deploy manually as a pod.
+
+* [Build 3scale operator image](#building3scaleoperatorimage)
+
+* Push image to public repo (for instance `quay.io`)
+
+```sh
+make push IMAGE=quay.io/myorg/3scale-operator VERSION=test
+```
+
+* Register the 3scale-operator CRDs in the OpenShift API Server
+
+```sh
+// As a cluster admin
+for i in `ls deploy/crds/*_crd.yaml`; do oc create -f $i ; done
+```
+
+* Create a new OpenShift project (optional)
+
+```sh
+export NAMESPACE=operator-test
+oc new-project $NAMESPACE
+```
+
+* Deploy the needed roles and ServiceAccounts
+
+```sh
+oc create -f deploy/service_account.yaml
+oc create -f deploy/role.yaml
+oc create -f deploy/role_binding.yaml
+```
+
+* Deploy the operator
+
+```sh
+sed -i 's|REPLACE_IMAGE|quay.io/myorg/3scale-operator:test|g' deploy/operator.yaml
+oc create -f deploy/operator.yaml
+```
+
+#### Cleanup manually deployed operator
+
+* Delete all `apimanager` custom resources:
+
+```sh
+oc delete apimanagers --all
+```
+
+* Delete the 3scale-operator operator, its associated roles and service accounts
+
+```sh
+oc delete -f deploy/operator.yaml
+oc delete -f deploy/role_binding.yaml
+oc delete -f deploy/service_account.yaml
+oc delete -f deploy/role.yaml
+```
+
+* Delete the APIManager CRD:
+
+```sh
+oc delete crds apimanagers.apps.3scale.net
+```
+
+### Deploy custom 3scale Operator using OLM
+
+To install this operator on OpenShift 4 for end-to-end testing,
+create the [Operator Source](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md#linking-the-quay-application-repository-to-your-openshift-40-cluster)
```

Comment:
It seems the anchor is broken.
We should explain that they need to first upload the bundle into a quay application repository (not sure if the original anchor directed to that first or it was just the creation of the operatorsource file)

Reply:
:+1:

Reply:
Anchor is working fine :thinking:

Reply:
Fixed!